### PR TITLE
docs: split README, wording

### PR DESCRIPTION
**Pull request by tsvetomir**
The removed section should go into the how-to section of the Kendo UI docs

### CODE REVIEW

**Comment by dmihaylo** [comment]
Made a couple of changes - please let me know if we need further updates in the article.

File: README.md [hunk 1]
```diff
@@ -1,70 +1,81 @@
...
-- All fonts will be loaded, even if only a few are needed.
-
-- Loading the generated `<script>` spends, on average, 15 milliseconds per font, to create a binary `Blob` from BASE64.  This should be faster than the classic method - loading the `.ttf` files as usual with plain CSS.
-
-## Browser support
-
-The generated bundle should work in all browsers that support [`Blob`](https://developer.mozilla.org/en/docs/Web/API/Blob).  Tested on IE 10, Firefox, Chrome.
-
-A workaround should be possible for IE9, but it's not yet implemented.
+The team are also working on the implementation of a workaround for Internet Explorer 9.
```
Comment (tsvetomir) [inline README.md]: Technically not true, should be removed.

**Comment by tsvetomir** [comment]
Apart from that last line it looks good :)

**Comment by dmihaylo** [comment]
@tsvetomir Done :) Thanks!

**Pull request merged**